src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -312,20 +316,22 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
 
             docker {
                 name '${id}'
-                tags 'latest', 'another'
+                tags 'mytag', 'another'
```

Comment:
I had this passing on Docker < 1.10 (the CI system ran 1.9 until very very recently), so not sure why this fails. Any chance you could walk through the error you’re encountering?